person.rb

```diff
@@ -1,17 +1,18 @@
 require_relative './nameable'
 require_relative './capitalize'
-require_relative './decorator'
+require_relative './rental'
 require_relative './trimmer'
 
 class Person < Nameable
   attr_accessor :name, :age, :id
-  attr_reader :parent_permission
+  attr_reader :parent_permission, :rentals
 
   def initialize(name = 'Unknown', age = 0, parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
     @id = rand(1..1000)
+    @rentals = []
     super()
```

Comment:
- [ ] Please add a method for adding rentals. Link to the [example](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) in a previous lesson.